ports/butterflysoup/port.json

```
@@ -0,0 +1,36 @@
+{
+  "version": 3,
+  "name": "butterflysoup.zip",
+  "items": [
+    "Butterfly Soup.sh",
+    "butterflysoup"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Butterfly Soup",
+    "porter": [
+      "all_yall"
+    ],
+    "desc": "A visual novel about gay asian girls playing baseball and falling in love.",
+    "desc_md": "A visual novel about gay asian girls playing baseball and falling in love.",
```

This is only necessary if you want to include markdown for the websites version of the description.
```suggestion
    "desc_md": null,
```